docs/api.md

```diff
@@ -10,6 +10,7 @@
 - [POST /api/auth/reactivate](##post-apiauthreactivate): Request a new activation code.
 - [POST /api/auth/forgot_password](#post-apiauthforgot_password): Request reset password link.
 - [GET /api/user_info](#get-apiuser_info): Get user's information.
+- [PATCH /api/sudo](#get-apiuser_sudo): Enable sudo mode.
```

Comment:
Nits: should `#get-apiuser_sudo` be `#patch-apiuser_sudo`?

Reply:
```suggestion
- [PATCH /api/sudo](#patch-apiuser_sudo): Enable sudo mode.
```